templates/patterns/carousel/carousel.ui_patterns.yml

```diff
@@ -1,77 +1,61 @@
+      label: 'Fade'
+      description: 'Animate slides with a fade transition instead of a slide. False by default.'
       preview: true
     with_controls:
       type: boolean
-      label: "With controls"
-      description: "With controls (default: false)."
+      label: 'With controls'
+      description: 'Enable the controls (prev and next buttons). False by default.'
       preview: true
-    prev_label:
-      type: select
-      label: "Previous label"
-      description: "The previous label value."
-      options:
-        1: "Previous"
-        2: "Previous slide"
-      preview: 1
-    next_label:
-      type: select
-      label: "Next label"
-      description: "The next label value."
-      options:
-        1: "Next"
-        2: "Next slide"
-      preview: 1
     with_indicators:
       type: boolean
-      label: "With indicators"
-      description: "With indicators (default: false)."
+      label: 'With indicators'
+      description: 'With indicators (default: false).'
       preview: true
     autoplay:
       type: boolean
-      label: "Disable autoplay"
-      description: "Disable auto slide movement."
+      label: 'Disable autoplay'
+      description: 'True by default. If set to false the carousel will not slide automatically.'
       preview: true
     autoinit:
       type: boolean
-      label: "Disable autoinit"
-      description: "Disable carousel autoinit on page load."
-      preview: false
+      label: 'initialize the carousel'
+      description: 'True by default. If set to false, the carousel will need to be instantiated via javascript.'
+      preview: true
     disable_touch:
       type: boolean
-      label: "Disable touch"
-      description: "Disable carousel swipe on touch."
+      label: 'Disable touch'
+      description: 'Allow the carousel to swipe on touch. False by default.'
+      preview: false
+    dark:
+      type: boolean
+      label: 'Dark'
+      description: 'Enable the dark mode.'
       preview: false
   fields:
-    active_item:
-      type: "text"
-      label: "Index of the active item"
     slides:
       type: array
-      label: "Slides"
-      description: "Each slide is a image with src and alt, caption, caption classes and interval of slide movement."
+      label: 'Slides'
+      description: 'Each slide is an image, a caption and interval of slide movement.'
       preview:
-        - caption: first slide label
-          caption_classes: "class-1 class-2"
+        - caption: 'First slide label'
           interval: 0
           image:
-            - theme: "image"
-              uri: "https://images.unsplash.com/photo-1533282960533-51328aa49826?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1140&h=385&q=80"
-              alt: "alt image 1"
+            - theme: 'image'
+              uri: 'https://images.unsplash.com/photo-1533282960533-51328aa49826?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1140&h=385&q=80'
```

Comment:
@escuriola, we should check together, there are 3 different web providers for images. Maybe we can check and use only one. This is not a key change.